packages/twenty-server/src/engine/workspace-manager/demo-objects-prefill-data/seed-workspace-member-with-demo-data.ts

```diff
@@ -8,7 +8,7 @@ export const DEMO_SEED_WORKSPACE_MEMBER_IDS = {
   TIM: '20202020-1553-45c6-a028-5a9064cce07e',
 };
 
-export const workspaceMemberPrefillData = async (
+export const seedWorkspaceMemberWithDemoData = async (
```

Comment:
we move it out from prefill data, aren't we breaking workspace creation?

Reply:
I need to check

Reply:
Any update on this?

Reply:
It's ok